.eslintrc

```diff
@@ -42,7 +42,8 @@
     "no-console": "off",
     "no-restricted-syntax": "off",
     "@typescript-eslint/no-empty-function": "off",
-    "import/no-unresolved": "off"
+    "import/no-unresolved": "off",
+    "@typescript-eslint/no-use-before-define": "off"
```

Comment:
Turned it off because it was not respecting hoisting functionality and throwing errors of old code on it, I would personally suggest on removing this.